source/_posts/2020-10-17-supervisor-248.markdown

```diff
@@ -0,0 +1,72 @@
+---
+title: "Supervisor 248: Resolution center, Stability & Private container registries"
+description: "Resolution center, Stability & Private container registries"
+date: 2020-10-16 00:00:00
+date_formatted: "October 17, 2020"
+comments: true
+author: Joakim Sørensen
+author_twitter: ludeeus
+categories: Release-Notes
+og_image: /images/blog/2020-10-17-supervisor-248/social.png
+---
+
+The Supervisor generally does not have enough changes that we can fill a blogpost with each release, it also does not have a fixed release schedule, a new Supervisor updated is released when it's needed, but that does not mean that there is nothing going on in this part of the ecosystem! One month ago we put out the [The Supervisor joins the party][supervisor-joins-the-party] post that went over the highlights from the past year. Now it’s time for version 248, rolling out to everyone as we speak and it’s full of new goodies, enjoy!
+
+## Resolution center
+
+We are introducing a new resolution center. This new feature will be used to identify issues and provide context and suggestions on solving the issues within your installation. We have big plans for this one!
+
+New in this release is that if the available free space on your device drops below 1 GB, auto-updates will be paused and you will be notified. This should give you time to clean it up.
+
+To help you free up some space, we have added a [new page to our documentation][more-info_free-space] that will help guide you through that.
+
+If you run Home Assistant Supervised, the system can get marked as unsupported if the host system does not have all the [required services running][required_services]. It was hard to find the reason for why a system is marked as unsupported. It is written to the logs during startup but the logs that you find in the UI do not go back that far. To make it more accessible, the message that your system is unsupported will now allow you to open a new dialog.
+
+![unsupported dialog](/images/blog/2020-10-17-supervisor-248/unsupported_dialog.png)
+
+This dialog will list all the reasons why the system was marked as unsupported. Clicking on the links in that dialog will take you to a documentation page that describes the issue, and there will be a solution you can use to make your installation compliant with [ADR0014] and [ADR0015].
+
+## Stability
+
+Stability is something that is always in focus for new Supervisor releases. The Supervisor is the main part of your supervised/Home Assistant OS installation, almost every release has some form of stability improvement, and this is not an exception to that.
+
+If Home Assistant fails to start, we will now rebuild the container. This will ensure that if something inside the container has been corrupted, a new fresh container is started up. We have also added this feature to our newly added [observer plugin][supervisor-joins-the-party-observer].
```

Comment:
```suggestion
If Home Assistant fails to start, we will now rebuild the container. This will ensure that a new fresh container is started up if something inside the container has been corrupted. We have also added this feature to our newly added [observer plugin][supervisor-joins-the-party-observer].
```